src/docs/respectful-code.md

```diff
@@ -0,0 +1,51 @@
+---
+title: "Respectful code"
```

Comment:
nit: we use single quotes elsewhere (both for `title` and `description`)

Reply:
Okay. The linter was complaining so I just ran `prettier` as I couldn’t find any linter fix scripts.